src/sentry/snuba/metrics/fields/base.py

```diff
@@ -17,12 +17,14 @@
 from snuba_sdk.orderby import Direction, OrderBy
 
 from sentry.api.utils import InvalidParams
-from sentry.models import Project, dataclass
+from sentry.models import Enum, Project, dataclass
```

Comment:
nit: import `Enum` and `dataclass` from std lib?

Reply:
oops